lib/adyen/api.rb

```diff
@@ -1,4 +1,3 @@
-require 'adyen'
```

Comment:
I had this here so you can `require 'adyen/api'` if you're not interested in the rest of the code base. I think this will fail without this require here. Same for `require 'adyen/rest'`